prime-router/src/test/kotlin/transport/RedoxTransportTests.kt

```diff
@@ -209,4 +209,51 @@ class RedoxTransportTests {
         assertEquals(1, retryItemsOut.size)
         assertEquals("1", retryItemsOut[0])
     }
+
```

Comment:
I thought of yet more tests to cover weird situations in the complex RedoxTransport code (already in production from a previous PR)